app/utils/deep_link/index.ts

```diff
@@ -88,21 +89,22 @@ export async function handleDeepLink(deepLinkUrl: string, intlShape?: IntlShape,
             }
             case DeepLink.Permalink: {
                 const deepLinkData = parsed.data as DeepLinkPermalink;
-                if (NavigationStore.hasModalsOpened() || ![Screens.HOME, Screens.CHANNEL, Screens.GLOBAL_THREADS, Screens.THREAD].includes(NavigationStore.getVisibleScreen())) {
+                if (NavigationStore.hasModalsOpened() ||
+                !([Screens.HOME, Screens.CHANNEL, Screens.GLOBAL_THREADS, Screens.THREAD] as AvailableScreens[]).includes(NavigationStore.getVisibleScreen())) {
                     await dismissAllModalsAndPopToRoot();
                 }
                 showPermalink(existingServerUrl, deepLinkData.teamName, deepLinkData.postId);
                 break;
             }
             case DeepLink.Plugin: {
-                const deepLinkData = parsed.data as DeepLinkPlugin;
-                showModal('PluginInternal', deepLinkData.id, {link: location});
+                // const deepLinkData = parsed.data as DeepLinkPlugin;
+                // showModal('PluginInternal', deepLinkData.id, {link: location});
```

Comment:
Any reason to leave this commented? We should either remove it, or create a todo with ticket associated.

Reply:
I'm not sure who owns this but the screen does not exist, so I commented the lines to remove the ts errors

Reply:
That should be integrations, but not sure this was ever properly implemented. Either way... if the screen does not exist, there may be a regression in stuff like connecting to plugins that require oauth (I think this screen was created in v1 for that purpose).

Reply:
@javaguirre can you create a ticket for this?

Reply:
https://mattermost.atlassian.net/browse/MM-49846